libs/perun/components/src/lib/group-resource-status/group-resource-status.component.html

```diff
@@ -2,23 +2,23 @@
   *ngIf="status === 'ACTIVE'"
   matTooltip="{{'SHARED_LIB.PERUN.COMPONENTS.GROUP_RESOURCE_STATUS.ACTIVE' | translate}}"
   matTooltipPosition="above">
-  <mat-icon>check_circle_outline</mat-icon>
+  <mat-icon [ngStyle]="{'color':'green'}">check_circle_outline</mat-icon>
```

Comment:
```suggestion
  <mat-icon class="green">check_circle_outline</mat-icon>
```
if you could use that sort of style it would be nice, it also moves the icon to the middle, the styles are defined in the `styles.scss` and you could also extend it to add colours that you need(black, grey)
![image](https://user-images.githubusercontent.com/32871167/126155918-2fdd878c-c01a-4327-892b-9e89bcc2d702.png)

Reply:
Done.